doc/sphinx/language/extensions/arguments-command.rst

```diff
@@ -50,51 +50,58 @@ Setting properties of a function's arguments
    it's not in a section.  Applying :attr:`global` within a section extends the
    effect outside the current sections and current module in which the command appears.
 
-      `/`
-         the function will be unfolded only if it's applied to at least the
-         arguments appearing before the `/`.  See :ref:`Args_effect_on_unfolding`.
+      :n:`@name {* % @scope }`
+         a *formal parameter* of the function :n:`@reference` (i.e.
+         the parameter name used in the function definition).  Unless `rename` is specified,
+         the list of :n:`@name`\s must be a prefix of the formal parameters, including all implicit
+         arguments.  `_` can be used to skip over a formal parameter.
+         This construct declares :n:`@name` as
+         non-implicit if `clear implicits` is specified or any
+         other :n:`@name` in the :cmd:`Arguments` command is declared implicit.
+         :token:`scope` can be either scope names or their delimiting
+         keys. When multiple scopes are present, notations are interpreted in the
+         leftmost scope containing them. See :ref:`binding_to_scope` for details.
 
-         .. exn:: The / modifier may only occur once.
+         .. exn:: To rename arguments the 'rename' flag must be specified.
             :undocumented:
```

Comment:
My mistake--this should have stayed where it was, but do include wording you added